go.mod

```diff
@@ -5,29 +5,22 @@ go 1.13
 require (
 	github.com/Masterminds/semver v1.5.0
 	github.com/alecthomas/kong v0.2.11
-	github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751 // indirect
 	github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d // indirect
 	github.com/crossplane/crossplane-runtime v0.12.0
```

Comment:
@khos2ow would you mind adding a replace to use @saschagrunert's fork from https://github.com/crossplane/crossplane-runtime/pull/210 so we can get this compiling?

Reply:
Not at all, will do in a minute.